src/initialize.test.js

```diff
@@ -279,14 +293,16 @@ describe('initialize', () => {
     expect(overrideHandlers.initError).toHaveBeenCalledWith(new Error('uhoh!'));
   });
 
-  it('should initialize the app with runtime configuration', async () => {
+  it('should initialize the app with runtime configuration by site', async () => {
     config.MFE_CONFIG_API_URL = 'http://localhost:18000/api/mfe/v1/config';
     config.APP_ID = 'auth';
     configureCache.mockReturnValueOnce(Promise.resolve({
       get: (url) => {
         const params = new URL(url).search;
         const mfe = new URLSearchParams(params).get('mfe');
-        return ({ data: { ...newConfig.common, ...newConfig[mfe] } });
+        const site = new URLSearchParams(params).get('site');
+        console.log(site);
```

Comment:
This `console.log` is necessary? @dcoa :eyes:

Reply:
Nop, my bad, I fix it, thanks Nico